### ISSUE TITLE
g200 test failure

### ISSUE THREAD

**Issue by joebo** [jsoftware/j-playground#36]
g200 test script fails. I can use some help interpreting what the case is doing and any theories of where to look

[Failing case](https://jsoftware.github.io/j-playground/bin/html2/#code=x%20%3D%3A%200.001%20*%20_900%2B%3F4%205%242000%0Ay%20%3D%3A%200.001%20*%20_900%2B%3F4%205%242000%0Aa%20%3D%3A%20x%20j.%20(*y)*(o.1)%7Cy%0Aa%20%3D%3A%20a%2C0%2C*%2F%5C4%240j1%0A%0Aexp%3D%3A%20(%25!i._50)%20%26%20(%2B%2F%20.*)%20%40%20((i._50)%26(%5E~%2F))%0A%0A5e_13%20%3E%20%7C%20(%5Ea)%25~(%5Ea)-exp%20a)

[Full script](https://jsoftware.github.io/j-playground/bin/html2/#url=https://raw.githubusercontent.com/jsoftware/j-playground/j903_wasm/test/g200.ijs)
```
   1:@:(dbr bind Debug)@:(9!:19)2^_44[(prolog [ echo^:ECHOFILENAME) './g200.ijs'
1
   NB. ^y ------------------------------------------------------------------
   
   x =: 0.001 * _900+?4 5$2000
   y =: 0.001 * _900+?4 5$2000
   a =: x j. (*y)*(o.1)|y
   a =: a,0,*/\4$0j1
   
   exp=: (%!i._50) & (+/ .*) @ ((i._50)&(^~/))
   
   5e_13 > | (^a)%~(^a)-exp a
0 0 0 0 0
0 0 0 0 0
0 0 0 0 0
0 0 0 0 0
0 0 0 0 0
|assertion failure: RUN1
|[-10] //test/g200.ijs

```

**Comment by joebo** [comment]
This test also fails on the linux_wasm build.  It will be easier to debug there

**Comment by joebo** [comment]
fixed with 517e00a2f9c77a2cbe65bd89d624331fb90fe563
